app/services/hawkular_proxy_service.rb

```diff
@@ -92,16 +92,31 @@ def tenants(limit)
     tenants = @cli.hawkular_client.http_get('/tenants')
 
     if @params['include'].blank?
-      tenants.map! { |x| x["id"] }
+      tenants.map! { |x| {:label => humanize(x["id"]), :value => x["id"]} }
     else
-      tenants.map! { |x| x["id"] if x["id"].include?(@params['include']) }
+      tenants.map! { |x| {:label => humanize(x["id"]), :value => x["id"]} if x["id"].include?(@params['include']) }
     end
 
     tenants.compact[0..limit]
   end
 
   private
 
+  def humanize(id)
+    special_cases = {
+      "_system" => "System",
+      "_ops"    => "Operations"
```

Comment:
@yaacov isn't this duplicating what you have in `container_live_dashboard_tenant_response.json`? Is there a way to unify this so we won't have multiple places to update?